btr-api/tests/mocks/significantIndividualsFiling/valid.json

```diff
@@ -1,55 +1,102 @@
 {
   "businessIdentifier": "12345",
-  "folioNumber": "67890",
-  "effectiveDate": "2022-01-01",
-  "significantIndividuals": [
+  "effectiveDate": "2023-03-01",
+  "entityStatement": {
+    "statementID": "bd4061d6-1a24-4356-93f3-c489b56610a2",
+    "statementType": "entityStatement",
+    "statementDate": "2023-02-28",
+    "isComponent": false,
+    "entityType": "registeredEntity",
+    "name": "Your Entity",
+    "identifiers": [],
+    "publicationDetails": {
+      "publicationDate": "2022-12-31",
+      "bodsVersion": "1.0",
+      "publisher": {
+        "uri": "http://publisher.uri",
+        "name": "Publisher Name",
+        "source": {
+          "url": "http://source.url"
+        }
+      }
+    },
+    "source": {
+      "source": "sample source",
+      "extractionDate": "YYYY-MM-DD"
```

Comment:
Shouldn't this fail validation? Should catch that this isn't a date right? Is this meant to be 'retrievedAt' ?

Reply:
This is some weird artefact, not sure how it ended up there, does not affect the schema/json at all. Will remove it. 
If it would be 'retrievedAt', yes, it would fail the verification.